apps/api-example/src/nestapp/app.module.ts

```diff
@@ -132,6 +134,35 @@ import {PollModule} from '@wepublish/poll/api'
       inject: [ConfigService],
       global: true
     }),
+    TrackingPixelModule.registerAsync({
+      imports: [ConfigModule],
+      useFactory: async (config: ConfigService) => {
+        const trackingPixelProviders: TrackingPixelProvider[] = []
+        const configFile = await readConfig(config.getOrThrow('CONFIG_FILE_PATH'))
+
+        const trackingPixelProvidersRaw = configFile.trackingPixelProviders
+        if (trackingPixelProvidersRaw) {
```

Comment:
Would love to remove parts of this heavy nesting with

`if (!trackingPixelProvidersRaw) {return trackingPixelProviders}`

And the same within the for-loop:
`if (trackingPixelProvider.type !== 'prolitteris') {continue}`